### ISSUE TITLE
path show with ipv6 loops forever and then is killed off

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#135]
**Describe the bug**

```
jpiet> path show src='2001:db8:0:1::1' dest='2001:db8:0:1::9' namespace=client                                                      Error: 1Killed
```

**Comment by jopietsch** [comment]
same thing is true for route lpm
```
jpiet> route lpm address='2001:db8:0:1::2'
```

**Comment by jopietsch** [comment]
with route lpm, it hangs at
```
        idx = df[['namespace', 'hostname', 'vrf', 'prefix']] \
            .query("prefix.ipnet.supernet_of('{}')".format(ipaddr)) \
            .groupby(by=['namespace', 'hostname', 'vrf'])['prefix'] \
            .max() \
```

**Comment by jopietsch** [comment]
also, I notice that all the v6 routes in the route table are /32, but in the network they are /64 or /128
```
jpiet> route show hostname=leaf01
   namespace hostname      vrf         prefix                                        nexthopIps          oifs protocol        source  metric  ipvers   action               timestamp
0     client   leaf01  default    10.0.1.2/32                        [169.254.0.1, 169.254.0.1]  [swp1, swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
1     client   leaf01  default    10.0.1.3/32                        [169.254.0.1, 169.254.0.1]  [swp1, swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
2     client   leaf01  default    10.0.1.4/32                        [169.254.0.1, 169.254.0.1]  [swp1, swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
3     client   leaf01  default    10.0.1.5/32                        [169.254.0.1, 169.254.0.1]  [swp1, swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
4     client   leaf01  default    10.0.1.6/32                        [169.254.0.1, 169.254.0.1]  [swp1, swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
5     client   leaf01  default   10.0.1.21/32                                     [169.254.0.1]        [swp1]      bgp                    20       4  forward 2020-06-05 21:40:29.243
6     client   leaf01  default   10.0.1.22/32                                     [169.254.0.1]        [swp2]      bgp                    20       4  forward 2020-06-05 21:40:29.243
7     client   leaf01  default  2001:db8::/32                          [fe80::4638:39ff:fe00:1]        [swp1]      bgp                    20       6  forward 2020-06-05 21:40:29.243
8     client   leaf01  default  2001:db8::/32                          [fe80::4638:39ff:fe00:d]        [swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
9     client   leaf01  default  2001:db8::/32  [fe80::4638:39ff:fe00:1, fe80::4638:39ff:fe00:d]  [swp1, swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
10    client   leaf01  default  2001:db8::/32  [fe80::4638:39ff:fe00:1, fe80::4638:39ff:fe00:d]  [swp1, swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
11    client   leaf01  default  2001:db8::/32  [fe80::4638:39ff:fe00:1, fe80::4638:39ff:fe00:d]  [swp1, swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
12    client   leaf01  default  2001:db8::/32  [fe80::4638:39ff:fe00:1, fe80::4638:39ff:fe00:d]  [swp1, swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
13    client   leaf01  default  2001:db8::/32  [fe80::4638:39ff:fe00:1, fe80::4638:39ff:fe00:d]  [swp1, swp2]      bgp                    20       6  forward 2020-06-05 21:40:29.243
14    client   leaf01     mgmt      0.0.0.0/0                                      [10.255.1.1]        [eth0]                             20       4  forward 2020-06-05 21:40:29.243
16    client   leaf01     mgmt  10.255.1.0/24                                                []        [eth0]   kernel  10.255.1.185      20       4  forward 2020-06-05 21:40:29.243
```

**Comment by ddutt** [comment]
Closed with the cyberpandas fix/checkin 95c1b4cd7c034ca4b35c62be497515b14dece3e4 and the checkins 087c2e7f336efdefa9b3767f37d4b1c8619b8879 and b04e273bc492ff5a0bcdac5253ff1e7a4bed8594
